client/src/helpers/eval-helper.js

```diff
@@ -121,7 +121,7 @@ class EvalHelperWorker {
   }
 
 
-  goToEhr (studentVisit) {
+  goToEhr (studentVisit, r = router) {
```

Comment:
I implemented a little change here in order to facilitate testing... I am not sure if this would be acceptable, but it's a quick way to test that the router function was called. See below for further explanation..

Reply:
are you using the router in the body of this function?  Can you use a longer name than just 'r'